flyteadmin_config.yaml

```diff
@@ -69,12 +69,17 @@ database:
 scheduler:
   eventScheduler:
     scheme: local
+    local:
     region: "my-region"
     scheduleRole: "arn:aws:iam::abc123:role/my-iam-role"
     targetName: "arn:aws:sqs:my-region:abc123:my-queue"
     scheduleNamePrefix: "flyte"
   workflowExecutor:
     scheme: local
+    local:
+      snapshotVersion: 1 # version of the snapshot
+      jitterValue: 30 # jitter or delta from the current time to use for finding the schedule
```

Comment:
can we sane default values for these

Reply:
and by default i mean, we should not need to add these values to our config at all!

Reply:
Moved jitter as a const in the scheduler